npm2deb/__init__.py

```diff
@@ -414,14 +418,24 @@ def read_package_info(self):
         self._get_json_version()
         self._get_json_license()
 
-    def download(self):
+    def download_tarball(self):
         utils.debug(1, "downloading %s@%s tarball from npm registry" % (self.name, self.version))
         info = _getstatusoutput('npm pack "%s@%s"' % (self.name, self.version))
         if info[0] is not 0:
             exception = "Error downloading package %s@%s\n" % (self.name, self.version)
             exception += info[1]
             raise ValueError(exception)
         tarball_file = info[1].split('\n')[-1]
+        extract_tarball(tarball_file)
+
+    def given_tarball(self)
+        utils.debug(1, "opening %s tarball" % (self.name))
```

Comment:
The function `given_tarball` is not being called anywhere

Reply:
This is what I get for working on this at random without keeping track of when I did what.
I wrote the code to avoid using a tarball, but it appears I never _called_ that code.    I thought I did, but I must have forgotten to commit it, or something similar.
Whoops.